<launch>
  <!-- joy node -->
  <node required = "true" pkg = "joy" type = "joy_node" name = "turtle_joy">
   <!-- <remap from = "joy" to = "cmd_vel" />-->
    <param name = "dev" type = "string" value = "/dev/input/js0"/>
    <param name = "deadzone" value = "0.12"/>
  </node>

  <!--axes-->
  <param name = "axis_linear_x" value = "3" type = "int" />
  <param name = "axis_linear_y" value = "2" type = "int" />
  <param name = "axis_angular_z" value = "0" type = "int" />
  <param name = "scale_linear_x" value = "0.3" type = "double" />
  <param name = "scale_linear_y" value = "0.3" type = "double" />
  <param name = "scale_angular_z" value = "1.0" type = "double" />
  
ã€€<node name = "teleop_car" pkg = "serial_hxb" type = "smartcar_joy"/> 

</launch>
